CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 
 ## 0.5.3dev
+* [Feature] Adds `%sqlcmd tables` (#76)
```

Comment:
Why don't we merge it with `%sql` ?

Reply:
`%sql` already has too many arguments and is too complex. I think we should group logic into different commands.